### PR TITLE
tuzov-forgot-password-back-button

### PR DESCRIPTION
**Pull request by pachkovska**
https://jira.pasv.us/browse/RBTA-32

includes back button tests on the forgot password page.

### CODE REVIEW

**Comment by artsenius** [comment]
> Tested it. It's working. You can also add a selector for the back button into the selector file and update the scripts accordingly.

You just asked the same as I asked you :)

File: test/specs/reg/forgot-password.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import exp from '../../expected/forgot-password';
+
+//Forgot password back button tests
+
+describe('Forgot Password Back Button', function () {
+
+    it('Back Button Text Font Size', function () {
+        browser.url('/');
+        $('#forgot').click();
+        let backButton = $('#back');
```
Comment (artsenius) [inline test/specs/reg/forgot-password.js]: Don't use '#forgot' and '#back' here. Store them in selectors and use as variable.

File: test/specs/reg/forgot-password.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        let fontSize = backButton.getCSSProperty('font-size').parsed.value;
+        assert.equal(fontSize, exp.fontSize);
+    });
+
+    it('Back Button Text Font Weight', function () {
+        browser.url('/');
+        $('#forgot').click();
```
Comment (artsenius) [inline test/specs/reg/forgot-password.js]: You don't have to refresh browser and click the button everytime. You can do that in the first test and then just verify the button.

File: test/specs/reg/forgot-password.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        let fontFamily = backButton.getCSSProperty('font-family').value;
+        assert.equal(fontFamily, exp.fontFamily);
+    });
+
+    it('Back Button Text Font Color', function () {
+        browser.url('/');
+        $('#forgot').click();
+        let backButton = $('#back');
+        let fontColor = backButton.getCSSProperty('color').parsed.hex;
+        console.log(fontColor);
```
Comment (artsenius) [inline test/specs/reg/forgot-password.js]: remove console log once everything works

File: test/specs/reg/forgot-password.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        assert.equal(fontColor, exp.fontColor);
+    });
+
+    it('Back Button Color', function () {
+        browser.url('/');
+        $('#forgot').click();
+        let backButton = $('#back');
+        let buttonColor = backButton.getCSSProperty('background-color').parsed.hex;
+        console.log(buttonColor);
```
Comment (artsenius) [inline test/specs/reg/forgot-password.js]: remove console log once everything works

File: test/specs/reg/forgot-password.js (hunk 3 above)
Comment (pachkovska) [reply test/specs/reg/forgot-password.js]: sorry, forgot to remove it

File: test/specs/reg/forgot-password.js (hunk 4 above)
Comment (pachkovska) [reply test/specs/reg/forgot-password.js]: yup. forgot))))

File: test/specs/reg/forgot-password.js (hunk 1 above)
Comment (pachkovska) [reply test/specs/reg/forgot-password.js]: refactored

File: test/specs/reg/forgot-password.js (hunk 2 above)
Comment (pachkovska) [reply test/specs/reg/forgot-password.js]: got it. ty. removed from all tests but one.

**Comment by itrusevich** [comment]
> > Tested it. It's working. You can also add a selector for the back button into the selector file and update the scripts accordingly.
> 
> You just asked the same as I asked you :)

@artsenius No, I only made a suggestion about one selector (back button). And I asked this before you commented on my PR:)

**Comment by artsenius** [comment]
Fix Merge conflicts

**Pull request merged**